sections/discussion/ourprocessreflections.tex

```diff
@@ -0,0 +1,78 @@
+\section{Reflections on our process}
+In \autoref{sec:our-process} we described the process we used to organize our project workflow and how we incorporated parts of the scrum framework to better fit our project
+In this section, we reflect on our process, and how it changed throughout the semester as well as what we would change for our future projects.
+
+\subsubsection{Scrum}
+Our Scrum inspired process worked well with sprints, sprint planning, sprint backlog, product owner, Scrum Master, planning poker, product backlog and product increment.
+We took turns on having the role as PO and Scrum Master.
+The PO/Scrum Master had the task of planning the next product backlog and add a DoD (definition of done) onto every task in the product backlog.
+This person also had the responsibility to send material to the supervisor, book a conference room and responsible for the planning poker.
+The sprint backlog and product backlog can be seen on \autoref{fig:trello-kanban}. 
+This is an kanban board on Trello we use to give an overview of how far people are with their tasks.
+The leftmost column is the product backlog, the next four column are the sprint backlog.
+These are split into \textit{Sprint Backlog}, \textit{In progress}, \textit{In review} and \textit{Done}.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.8\linewidth]{figures/trellopicture.PNG}
+    \caption{Our kanban board for the product and sprint backlog on Trello}
+    \label{fig:trello-kanban}
+\end{figure}
+\noindent
+When we conducted the sprint planning, we would use planning poker to estimate how many story points each task in the proposed sprint backlog should have.
+The persons with the highest and lowest story points would discuss the reasoning behind it.
+The story points were often very different if there were ambiguities, and people understood the tasks differently.
+This gave everyone the same understanding on how the given tasks should be solved and eliminate most ambiguities. 
+\\
+However, in the middle of the project when we started to develop the application and learning \textit{React}, it started to become less systematic.
+This was often due to it being difficult to estimate how long each task would take to solve.
+When we got more familiar with \textit{React} we were able to estimate how much the tasks took, and it got more systematic again.
+\\\\
+We also stopped conducting the retrospective meetings.
+The Scrum Masters stopped enforcing the retrospective meetings so they started fading out.
+These meetings could have helped making the project more structured as we could have discussed the problems that we faced and the reasons why we did not complete our tasks in time.
+
+\subsubsection{Formal reviews}
+The formal reviews was something that we changed in the middle of the semester, so that two people had to review a pull request together.
```

Comment:
```suggestion
The formal reviews were something that we changed in the middle of the semester, so that two people had to review a pull request together.
```